README.md

```diff
@@ -107,11 +107,20 @@ You probably need to set some environment variables for the client to be useful.
   - Disabled by default.
   - Set to `true` enable.
 
+## Troubleshooting
+
+### The UI shows more sensors than I have
+
+The firmware reads all 12 ADC pins on the Teensyduino. You may get fake readings from pins that have no sensors plugged in and are not grounded. If pressing a sensor causes more than one value to go up:
```

Comment:
Talking about "Teensyduino" is technically incorrect, since we actually don't use the Teensyduino. Just "Teensy" (or maybe "Teensy 2.0" as long as it's the only Teensy model supported) would be correct thing to use.

Reply:
Changed, thanks!